deploy/06_deployment.yaml

```diff
@@ -0,0 +1,51 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: openshift-lws-operator
+  namespace: openshift-lws-operator
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      name: openshift-lws-operator
+  template:
+    metadata:
+      labels:
+        name: openshift-lws-operator
+    spec:
+      securityContext:
+        runAsNonRoot: true
+        seccompProfile:
+          type: RuntimeDefault
+      containers:
+        - name: openshift-lws-operator
+          securityContext:
+            allowPrivilegeEscalation: false
+            readOnlyRootFilesystem: true
+            capabilities:
+              drop: [ "ALL" ]
+          image: quay.io/aguclu/lws-operator:test # quay.io/openshift/origin-lws-operator:latest
```

Comment:
This value will be replaced by bundle image.

Reply:
I am planning to use `quay.io/openshift/origin-lws-operator:latest`. This will be replaced by Konflux anyway. But I'll definitely not use from my quay account, just for testing for now.

Reply:
+1